boards/wio_terminal/src/buttons.rs

```diff
@@ -41,8 +41,15 @@ impl ButtonPins {
         let eic_clock = clocks.eic(&gclk1).unwrap();
         let mut eic = eic::init_with_ulp32k(mclk, eic_clock, eic);
 
-        /*
         eic.button_debounce_pins(&[
+            11 as ExternalInterruptID,
+            12 as ExternalInterruptID,
+            3 as ExternalInterruptID,
+            4 as ExternalInterruptID,
+            5 as ExternalInterruptID,
+            10 as ExternalInterruptID,
+            7 as ExternalInterruptID,
```

Comment:
I think I see the problem. The `ExternalInterrupt` trait is not defined in a reasonable way anymore. This is probably a problem introduced by the migration to `gpio::v2`. I think I have a way to fix it, so that you can use `.id()` again. Let me see if I can implement that first, before we resort to manually specifying all the `ExternalInterruptID`s

Reply:
@bradleyharden That'd be great; thanks.  I hope to be back on wio/rust by the end of the week, when I'll take another stab at #628 and can update this too.